packages/next/client/image.tsx

```diff
@@ -151,6 +178,7 @@ type GenImgAttrsData = {
   unoptimized: boolean
   layout: LayoutValue
   loader: ImageLoader
+  isStatic?: boolean
```

Comment:
Do we need both `isStatic` and `meta`? Perhaps `ImageMetaData` should have `isStatic`?